services/QuillLMS/client/app/bundles/Teacher/components/scorebook/score_legend.jsx

```diff
@@ -1,63 +1,76 @@
+            <p className="explanation">{explanation}</p>
+          </div>
+        </div>
+      }
+    />
+
+  )
+}
+
 export default class ScoreLegend extends React.Component {
   render() {
     const cutOff = proficiencyCutoffsAsPercentage();
     return (
       <div className="icons-wrapper icon-legend score-legend">
         <div className="icons">
-          <div className="icon">
-            <div className="icon-wrapper icon-green" />
-            <div className="icons-description-wrapper">
-              <p className="title">Frequently demonstrated skill</p>
-              <p className="explanation">{`100 - ${cutOff.proficient}% of prompts exhibit skill`}</p>
-            </div>
-          </div>
-          <div className="icon">
-            <div className="icon-wrapper icon-orange" />
-            <div className="icons-description-wrapper">
-              <p className="title">Sometimes demonstrated skill</p>
-              <p className="explanation">{`${cutOff.proficient - 1} - ${cutOff.nearlyProficient}% of prompts exhibit skill`}</p>
-            </div>
-          </div>
-          <div className="icon">
-            <div className="icon-wrapper icon-red" />
-            <div className="icons-description-wrapper">
-              <p className="title">Rarely demonstrated skill</p>
-              <p className="explanation">{`${cutOff.nearlyProficient - 1} - 0% of prompts exhibit skill`}</p>
-            </div>
-          </div>
-          <Tooltip
-            tooltipText='This type of activity is not graded.'
-            tooltipTriggerText={
-              <div className="icon">
-                <div className="icon-wrapper icon-blue" />
-                <div className="icons-description-wrapper">
-                  <p className="title">Completed</p>
-                  <p className="explanation">No score provided</p>
-                </div>
+          <Icon
+            explanation={`100 - ${cutOff.proficient}% of prompts exhibit skill`}
+            icon={<div className="icon-wrapper icon-green" />}
+            title={<span>Frequently<br />Demonstrated Skill</span>}
+            tooltipText={tooltipCopyForScoreDescriptions['frequently demonstrated skill']}
+          />
+          <Icon
+            explanation={`${cutOff.proficient - 1} - ${cutOff.nearlyProficient}% of prompts exhibit skill`}
+            icon={<div className="icon-wrapper icon-orange" />}
+            title={<span>Sometimes<br />Demonstrated Skill</span>}
+            tooltipText={tooltipCopyForScoreDescriptions['sometimes demonstrated skill']}
+          />
+          <Icon
+            explanation={`${cutOff.nearlyProficient - 1} - 0% of prompts exhibit skill`}
+            icon={<div className="icon-wrapper icon-red" />}
+            title={<span>Rarely<br />Demonstrated Skill</span>}
+            tooltipText={tooltipCopyForScoreDescriptions['rarely demonstrated skill']}
```

Comment:
I think we use "frequently demonstrated skill", "sometimes demonstrated skill", and "rarely demonstrated skill" in several other places around the codebase so it might be time to constantize them in the Shared bundle?

Reply:
Yeah, the casing is different here but your point stands. I added a comment to this existing clean-up card to take care of that: https://www.notion.so/quill/Clean-up-Task-Update-codebase-to-move-away-from-variable-names-around-proficiency-vs-demonstrated--f53d3189c27549f99febaf95b7df288a?pvs=4.